src/backend/metal/src/command.rs

```diff
@@ -759,12 +759,29 @@ impl<'a> PreRender<'a> {
             PreRender::Void => (),
         }
     }
+
+    fn issue_many<'b, I>(&mut self, commands: I)
```

Comment:
Not really important for now, but it seems like we should be able to collapse these functions by being generic over the command type